opentelemetry-sdk/src/opentelemetry/sdk/environment_variables/__init__.py

```diff
@@ -140,16 +183,28 @@
 OTEL_EXPORTER_OTLP_PROTOCOL = "OTEL_EXPORTER_OTLP_PROTOCOL"
```

Comment:
Missing Zipkin?

Reply:
Hey Leighton, not sure what I'm missing here (Zipkin). I plan to address all the other changes mentioned.

The following Zipkin variables are defined:
- OTEL_EXPORTER_ZIPKIN_ENDPOINT
- OTEL_EXPORTER_ZIPKIN_TIMEOUT

Reply:
My mistake, must've missed these :)